app/screens/ContactPickerScreen.tsx

```diff
@@ -117,17 +116,20 @@ export const ContactPickerScreen: FC<ContactPickerScreenProps> = observer(
     }
 
     const addCustomContact = async (data) => {
-      let pubkey = data.pubkey
+      let pubkey = data.pubkey.trim()
```

Comment:
this function should be this:

```
  const addCustomContact = async () => {
    let pubkey: string = customContact.trim()
    try {
      pubkey = await resolvePubkey(pubkey)
    } catch (e) {
      alert(`Invalid contact: ${e}`)
    }
    try {
      addContact({ pubkey, legacy: true, secret: false }, mgr)
      navigation.goBack()
    } catch (e) {
      alert(`Invalid contact: ${e}`)
    }
  }
```

if you do not call addContact, it will not be saved for real.  it will only be in mobx state tree, and if you log out and log in again, you will lose all your contacts